Dockerfile

```diff
@@ -14,11 +14,17 @@ ENV APP_URL=http://accounts.jenkins.io/
 EXPOSE 8080
 
 USER root
+RUN \
+  apk add --no-cache git python  && \
+  mkdir -p /opt/stv && \
+  git clone https://github.com/louridas/stv.git /opt/stv
```

Comment:
I think this should clone/checkout a specific SHA1 to ensure repeatability of these builds